Tests/SnapshotConfiguration.swift

```diff
@@ -68,6 +68,10 @@ extension SnapshotConfiguration {
     "Issues/issue_1664": .customValueProviders([
       AnimationKeypath(keypath: "**.base_color.**.Color"): ColorValueProvider(.black),
     ]),
+    
+    "Issues/issue_1847": .customValueProviders([
+      AnimationKeypath(keypath: "**.Stroke 1.**.Color"): ColorValueProvider(.red),
```

Comment:
Before this PR, this value provider was unexpectedly only applied when using the Main thread rendering engine but not when using the Core Animation rendering engine.